sys/shell/commands/sc_pm.c

```diff
@@ -107,6 +106,7 @@ static int cmd_unblock(char *arg)
         return 1;
     }
 
+    pm_blocker = pm_get_blocker();
```

Comment:
here and below: needs declaration? well, murdock will complain if it does.

Reply:
Indeed. I amended the fix right into the commit.